### DISCUSSION TITLE
找不到 Lively.Ul.WinUlexe

### DISCUSSION THREAD

**Discussion by whbook** [rocksdanister/lively#2057] [Q&A]
EXCEPTION:
An error occurred trying to start process'DA我的文档
\Git\lively\src\Lively\Lively\bin\Debug\net7.0-windows10.0.18362\plugins\UN\Lively.Ul.WinUlexe'with working directory 'D:\文档
\Git\lively\src\Lively\Lively\bin\Debug\net7.0-windows10.0.18362 plugins\U'.系统找不到指定的文件。

我在visual studio 2022上尝试运行时，出现以上错误。

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/lively/wiki/Building
